packages/runtimes/js/src/JSRuntime.js

```diff
@@ -185,12 +185,24 @@ function getURLRuntime(
   bundle: Bundle,
   externalBundle: Bundle,
 ): RuntimeAsset {
+  let relativePath = relativeBundlePath(bundle, externalBundle, {
+    leadingDotSlash: false,
+  });
+
+  if (dependency.meta.webworker === true) {
```

Comment:
Maybe `dependency.env.context === 'web-worker'` instead?

Reply:
I considered this. Unfortunately there's no way that I found to identify a dependency on a new worker bundle entry (they're not entry bundles unlike service workers). Any alternative?

Reply:
Ah, maybe if the dependency's environment is web worker but not the parent bundle?

Reply:
If a worker invokes `new Worker()`, that new worker will also need the runtime, unfortunately. We need to specifically detect if the dependency was created from `new Worker()`, and I'm not sure it's doable in another way.

If we continue with the meta property, the name for the property could be better, though.